doc/api/cli.md

```diff
@@ -135,16 +180,25 @@ see those as two separate modules and would attempt to load the module multiple
 times, causing an exception to be thrown).
 
 ### `--track-heap-objects`
+<!-- YAML
+added: v2.4.0
+-->
 
 Track heap object allocations for heap snapshots.
 
 
 ### `--prof-process`
+<!-- YAML
+added: v6.0.0
```

Comment:
Here's the commit where it was added: https://github.com/nodejs/node/commit/49440b7ea2496dbb7b343cf1872d96b92be470db

v6.0.0